src/ProjectAudioManager.cpp

```diff
@@ -1089,10 +1076,18 @@ void ProjectAudioManager::OnCommitRecording()
 
 void ProjectAudioManager::OnSoundActivationThreshold()
 {
-   auto &project = mProject;
    auto gAudioIO = AudioIO::Get();
-   if ( gAudioIO && &project == gAudioIO->GetOwningProject().get() ) {
```

Comment:
The simplification lost the restriction, testing GetOwningProject.  What justifies the removal of that condition?

Reply:
Good point. Nothing justifies it, it was my mistake. I will reinstate the check.